API.md

```diff
@@ -1421,7 +1421,7 @@ Alternatives can be expressed using the shorter `[]` notation.
 
 ```javascript
 var alt = Joi.alternatives().try(Joi.number(), Joi.string());
-// Same as [Joi.number(), Joi.string()]
+// Same as Joi.alternatives([Joi.number(), Joi.string()])
```

Comment:
OK, here's my latest proposal:


### `alternatives`

Generates a type that will match one of the provided alternative schemas via the [`try()`](#alternativestryschemas)
method. If no schemas are added, the type will not match any value except for `undefined`.

Supports the same methods of the [`any()`](#any) type.

#### `alternatives.try(schema, schema2, ...| [schema, schema2, ...])`

Try accepts schemas using single or multiple arguments, or as an array.

```javascript
var alt = Joi.alternatives().try(Joi.number(), Joi.string());
alt.validate('a', function (err, value) { });
```

Using `.try()` is optional and allows for a shorter version:

```javascript
var alt = Joi.alternatives(Joi.number(), Joi.string())
```